Framework/Algorithms/src/FitPeak.cpp

```diff
@@ -744,6 +744,8 @@ double FitOneSinglePeak::fitFunctionMD(IFunction_sptr fitfunc,
   boost::shared_ptr<MultiDomainFunction> funcmd =
       boost::make_shared<MultiDomainFunction>();
 
+  funcmd->setAttributeValue("NumDeriv", false);
```

Comment:
Please explain why you are setting NumDeriv to false, perhaps also an explanation why you changed the default in the constructor in MultiDomainFunction.cpp to true. thanks